smartsim/experiment.py

```diff
@@ -868,49 +868,48 @@ def reconnect_orchestrator(self, checkpoint: str) -> Orchestrator:
     def preview(
         self,
         *args: t.Any,
-        output_format: previewrenderer.Format = previewrenderer.Format.PLAINTEXT,
         verbosity_level: previewrenderer.Verbosity = previewrenderer.Verbosity.INFO,
+        output_format: previewrenderer.Format = previewrenderer.Format.PLAINTEXT,
```

Comment:
Your additions, unfortunately, cause `experiment.py` to go over our 1k LoC limit per file. Let's not up the global limit, yet. Please add:

```py
# pylint: disable=too-many-lines
```

See [`_core/_install/builder.py`](https://github.com/CrayLabs/SmartSim/blob/develop/smartsim/_core/_install/builder.py) for example

Reply:
changed